CMakeLists.txt

```diff
@@ -22,10 +24,14 @@ target_include_directories(Plugin_TheoraVideoSystem PUBLIC include/)
 target_link_libraries(Plugin_TheoraVideoSystem PUBLIC theoraplayer OgreMain)
 set_target_properties(Plugin_TheoraVideoSystem PROPERTIES PREFIX "")
 
-add_executable(player demos/player/player.cpp)
-target_link_libraries(player OgreBites Plugin_TheoraVideoSystem)
-
-file(COPY demos/resources.cfg DESTINATION ${CMAKE_BINARY_DIR})
+if (BUILD_DEMOS)
+	find_package(OGRE REQUIRED COMPONENTS Bites)
```

Comment:
instead of searching a second time here, remove the required part above, but check `OGRE_Bites_FOUND ` here. see: https://ogrecave.github.io/ogre/api/latest/setup.html